dev/ansible/04_test_persistence.yml

```diff
@@ -0,0 +1,182 @@
+# This file contains checks and test scenarios that test the persistence feature for postgres
+- name: set variable
+  set_fact:
+    consul_token: "master"
+  tags: test
+
+- name: Fetch bootstrap token nomad
+  shell:
+    cmd: vault kv get -field=secret-id secret/nomad-bootstrap-token
+  when: lookup('env', 'nomad_acl') | bool
+  register: bootstrap_token
+  tags: test
+
+- name: Verify if host volume has been added on nomad node(acl enabled)
+  shell:
+    cmd: nomad node status -short -self | grep 'Host Volumes' | tr -d " \t\n\r"  | cut -d "=" -f2
+  register: nomad_node_status
+  when: lookup('env', 'nomad_acl') | bool
+  failed_when: nomad_node_status.stdout != "persistence"
+  environment:
+    NOMAD_TOKEN: "{{ bootstrap_token.stdout }}"
+  tags: test
+
+- name: Verify if host volume has been added on nomad node(acl disabled)
+  shell:
+    cmd: nomad node status -short -self | grep 'Host Volumes' | tr -d " \t\n\r"  | cut -d "=" -f2
+  register: nomad_node_status_1
+  failed_when: nomad_node_status_1.stdout != "persistence"
```

Comment:
we have persistence as default, but it is entirely possible to give another host volume name. This would fail in that case. Could we somehow use a variable here instead? I dont have a very clear solution in mind

Reply:
If we change it to be a variable and not default persistence there need to be other code changes throughout. If we want to change it I think we should make it as a separate issue.

Reply:
Cool, let's just do that for now then. Important that we create an issue, so we don't forget!

Reply:
New issue #28